src/libfido2Config.cmake.in

```diff
@@ -0,0 +1,10 @@
+set(FIDO_VERSION @FIDO_VERSION@)
+
+@PACKAGE_INIT@
+
+set_and_check(FIDO_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+set_and_check(FIDO_INCLUDE_DIRS "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
```

Comment:
Are these necessary given the exported targets (maybe needs `INCLUDES DESTINATION`)?

Reply:
hmm, indeed. https://cmake.org/cmake/help/latest/prop_tgt/INTERFACE_INCLUDE_DIRECTORIES.html seems to be what should be set for the target